plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java

```diff
@@ -14,32 +14,24 @@
 package io.trino.plugin.bigquery;
 
 import io.trino.testing.sql.SqlExecutor;
-import io.trino.testing.sql.TestTable;
-
-import java.util.List;
+import io.trino.testing.sql.TemporaryRelation;
 
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class BigQueryTestView
-        extends TestTable
+        implements TemporaryRelation
 {
-    private final TestTable table;
+    private final SqlExecutor sqlExecutor;
+    private final TemporaryRelation table;
     private final String viewName;
 
-    public BigQueryTestView(SqlExecutor sqlExecutor, TestTable table)
+    public BigQueryTestView(SqlExecutor sqlExecutor, TemporaryRelation table)
```

Comment:
nit/optional: rename `table` to `relation`

Reply:
thanks, renaming in the commit in which we no longer assume this is a table.